test/payment/v2/payment-mappers/payment-intent-mapper.spec.js

```diff
@@ -0,0 +1,47 @@
+import GatewayMapper from '../../../../src/payment/v2/payment-mappers/gateway-mapper';
+import QuoteMapper from '../../../../src/payment/v2/payment-mappers/quote-mapper';
```

Comment:
If possible (I meant if there is the exported index) please get together this imports, because is part of the same path
```suggestion
import { GatewayMapper , StoreMapper , QuoteMapper } from '../../../../src/payment/v2/payment-mappers';
```

Reply:
Is not possible, I needed to put one by one